### DISCUSSION TITLE
is a WLAN (client mode) enable/disable via rfid possible?

### DISCUSSION THREAD

**Discussion by kugelzucker** [MiczFlor/RPi-Jukebox-RFID#1956] [Ideas]
(i tried searching for this, sorry if dupe)

i finished a phoniebox-build (only procrastinated for a year) this week and now i have one question regarding WLAN.

i am using future3/main (3.2.1) and everything works perfectly, but i cant find a RPC command that i could use with an RFID tag to enable/disale WLAN.

to be clear: i am not talking about the autohotspot function but the box connection in client mode to a known AP on demand. 

use case is maintenance and uploading new media when needed (i do that at home, where my wifi is). i want to disable the wifi after this to save some energy, reduce attack surface (unlikely, i know) and most importantly: reduce radiation (SCNR).

**Comment by s-martin** [comment]
Hi, while this feature exists in 2.x it’s not available in Version 3, see https://rpi-jukebox-rfid.readthedocs.io/en/latest/featurelist.html

**Comment by pabera** [comment]
@s-martin is right. But it should be to hard to implement, though ;-)

I imagine it could go somewhere [here](https://github.com/MiczFlor/RPi-Jukebox-RFID/blob/f56c36978ac1f5219ebdc1a51bd62fe53edb8ae0/src/jukebox/components/hostif/linux/__init__.py)

**Reply by kugelzucker** [reply]
i guess i have to learn python now. at least more than my hello-world-knowledge. BRB.

**Reply by kugelzucker** [reply]
your pointer seems to be the right place. and from what i can see it seems doable at my skilllevel. ill try it and ping you, please point out stupid mistakes :)
